project/src/main/ui/settings/VolumeSetting.tscn

```diff
@@ -55,6 +56,9 @@ max_value = 1.0
 step = 0.01
 tick_count = 11
 
+[node name="HSliderGamepadHelper" parent="HBoxContainer/HSlider" instance=ExtResource( 5 )]
+sliding_speed = 0.5
```

Comment:
It would be good to capture the 'sliding speed' setting logic somewhere, just in a comment maybe.

Reply:
Added comment; `for example, our HSV sliders have a length of 256, and their speed is 128.`